### PR TITLE
Fix linter deprecation warnings/errors for DDP-SDK and BASIL-APP

### PR DESCRIPTION
**Pull request by alextanlan**

### CODE REVIEW

File: ddp-workspace/projects/ddp-sdk/src/lib/directives/addressGoogleAutocomplete.directive.ts [hunk 1]
```diff
@@ -47,9 +47,13 @@ export class AddressGoogleAutocompleteDirective implements OnInit, OnDestroy, On
...
             },
-            () => this.logger.logWarning(this.LOG_SOURCE, 'Could not load google-maps-places script.'));
+            () => this.logger.logWarning(this.LOG_SOURCE, 'Could not load google-maps-places script.')
+        );
         // making sure that any countryCodes handled AFTER scriptloader has processed
-        this.scriptLoader$.pipe(concat(this.countryCode$), skip(1), takeUntil(this.ngUnsubscribe)).subscribe((countryCode) => {
+        concat(this.scriptLoader$, this.countryCode$).pipe(
+            skip(1),
+            takeUntil(this.ngUnsubscribe)
+        ).subscribe((countryCode) => {
```
Comment (MocanaAtBroad) [inline ddp-workspace/projects/ddp-sdk/src/lib/directives/addressGoogleAutocomplete.directive.ts]: Getting a lint warning on unused parameter `countryCode`. I think you can safely remove it.

**Review by MocanaAtBroad — DISMISSED** [review]

File: ddp-workspace/projects/ddp-sdk/src/lib/directives/addressGoogleAutocomplete.directive.ts (hunk 1 above)
Comment (alextanlan) [reply ddp-workspace/projects/ddp-sdk/src/lib/directives/addressGoogleAutocomplete.directive.ts]: Fixed.

**Comment by softcat23** [comment]
Nice job! But getting 2 lint warnings here:
```
/ddp-angular/ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts:33:11
WARNING: 33:11  deprecation  combineLatest is deprecated: Deprecated in favor of static combineLatest.
WARNING: 36:11  deprecation  flatMap is deprecated: resultSelector no longer supported, use inner map instead
```

**Comment by fat23cat** [comment]
Nice job! But getting 2 lint warnings here:
```
/ddp-angular/ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/activityServiceAgent.service.ts:33:11
WARNING: 33:11  deprecation  combineLatest is deprecated: Deprecated in favor of static combineLatest.
WARNING: 36:11  deprecation  flatMap is deprecated: resultSelector no longer supported, use inner map instead
```

**Comment by alextanlan** [comment]
You are right. They were fixed later in  f15d8a5cea1c17940810979238b48eb746e73c5d .

**Review by fat23cat — DISMISSED** [review]

**Review by MocanaAtBroad — APPROVED** [review]

**Pull request merged**